components/TextFields/examples/TextFieldCustomFontExample.m

```diff
@@ -393,18 +426,18 @@ - (BOOL)isValidPhoneNumber:(NSString *)inputString partially:(BOOL)isPartialChec
 
 @end
 
-@implementation TextFieldDynamicSizeExample (CatalogByConvention)
+@implementation TextFieldCustomFontExample (CatalogByConvention)
 
 + (NSArray *)catalogBreadcrumbs {
-  return @[ @"Text Field", @"Dynamic Size Fonts (Objective-C)" ];
+  return @[ @"Text Field", @"Custom Fonts (Objective-C)" ];
 }
 
 + (BOOL)catalogIsPrimaryDemo {
   return NO;
 }
 
 + (BOOL)catalogIsPresentable {
-  return YES;
+  return NO;
```

Comment:
What does this do again?

Reply:
It's only shown in Dragon but not Catalog.